python/tvm/relay/frontend/onnx.py

```diff
@@ -688,13 +691,28 @@ def _impl_v11(cls, inputs, attr, params):
                 output_padding = attr.get("output_padding", [0] * kndim)
                 strides = attr["strides"]
                 total_pad = [0] * kndim
-                for i in range(kndim):
-                    total_pad[i] = (
-                        output_padding[i] + ((kernel_shape[i] - 1) * dilations[i] + 1) - strides[i]
-                    )
+                if "output_shape" in attr:
```

Comment:
Can you post a reference back to the docs on how these are calculated? https://github.com/onnx/onnx/blob/main/docs/Operators.md#ConvTranspose